emulate-browsers/chrome-80/index.ts

```diff
@@ -5,33 +5,29 @@ import {
   DomOverridesBuilder,
   getEngineExecutablePath,
   getTcpSettingsForOs,
-  IUserAgent,
   modifyHeaders,
   parseNavigatorPlugins,
-  readPolyfills,
-  StatcounterBrowserUsage,
+  DataLoader,
+  DomDiffLoader,
 } from '@secret-agent/emulate-browsers-base';
+import IUserAgentOption from '@secret-agent/emulate-browsers-base/interfaces/IUserAgentOption';
 import { randomBytes } from 'crypto';
 import IUserProfile from '@secret-agent/core-interfaces/IUserProfile';
-import UserAgents from '@secret-agent/emulate-browsers-base/lib/UserAgents';
 import { pickRandom } from '@secret-agent/commons/utils';
-import navigator from './navigator.json';
-import chrome from './chrome.json';
-import codecs from './codecs.json';
 import pkg from './package.json';
-import headerProfiles from './headers.json';
-import defaultUseragents from './user-agents.json';
-import frame from './frame.json';
+import headerProfiles from './data/headers.json';
+import userAgentOptions from './data/user-agent-options.json';
 
-const polyfillSet = readPolyfills(__dirname);
-const agents = UserAgents.getSupportedAgents('Chrome', 80, defaultUseragents);
+const windowFramingData = new DataLoader(`${__dirname}/data`, 'window-framing');
+const windowChromeData = new DataLoader(`${__dirname}/data`, 'window-chrome');
+const windowNavigatorData = new DataLoader(`${__dirname}/data`, 'window-navigator');
+const codecsData = new DataLoader(`${__dirname}/data`, 'codecs');
+const domDiffsData = new DomDiffLoader(`${__dirname}/data`);
 
 @BrowserEmulatorClassDecorator
 export default class Chrome80 {
   public static id = pkg.name;
-  public static roundRobinPercent = StatcounterBrowserUsage.getConsumerUsageForBrowser(
-    'Chrome 80.0',
-  );
+  public static roundRobinPercent = 0; // TODO; Import from json
```

Comment:
Are you planning to import the consumer browser usage from double agent?

Reply:
Shit. Yes.

Reply:
Fixed.